controllers/single_page/dashboard/store/settings.php

```diff
@@ -235,6 +235,9 @@ public function save()
                 Config::save('community_store::products.hideSize', !empty($args['hideSize']));
                 Config::save('community_store::products.hideWeight', !empty($args['hideWeight']));
                 Config::save('community_store::products.hideBarcode', !empty($args['hideBarcode']));
+                
+                Config::save('community_store::products.newVariantDisable', boolval($args['newVariantDisable']));
```

Comment:
In PHP, accessing undefined array indexes raises a warning.
This can be avoided by using `empty()`, which accepts non-existing array indexes.

So, I'd change from

```php
boolval($args['newVariantDisable'])
```

To
```php
!empty($args['newVariantDisable'])
```

(And PS: generally speaking, `(bool) ...` is much faster than `boolval(...)`)

Reply:
I've done the change.